frontend/.env.example

```
@@ -3,6 +3,7 @@ EXPO_PUBLIC_REDIRECT_URI=<redirect-uri>            # Redirect URI for authentica
 EXPO_PUBLIC_TOKEN_URL=https://api.asgardeo.io/t/<asgardeo-organization>/oauth2/token
 EXPO_PUBLIC_LOGOUT_URL=https://api.asgardeo.io/t/<asgardeo-organization>/oidc/logout
 EXPO_PUBLIC_BACKEND_BASE_URL=<backend-url>         # Backend API Base URL
+EXPO_PUBLIC_CHAT_AGENT_URL=<chat-agent-url>        # Chat agent API base URL (e.g. http://localhost:8000)
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The EXPO_PUBLIC_CHAT_AGENT_URL key should go before the EXPO_PUBLIC_CLIENT_ID key

(UnorderedKey)